app/controllers/user/mygardens/plants_controller.rb

```diff
@@ -6,8 +6,23 @@ def index
 	def create
 		garden = Garden.find(params[:garden_id])
 		plant = Plant.find(params[:plant_id])
-		garden.plants << plant
-		garden.save
+    garden.plants << plant
+    garden.save
+    add_notification(garden, plant)
```

Comment:
an `event` is added to the `calendar` when the `user` adds a `plant` to the `garden